--- autotrace-0.31.10/Makefile.am.orig	2024-04-24 20:13:30.403400098 +0200
+++ autotrace-0.31.10/Makefile.am	2024-04-24 20:21:31.257461755 +0200
@@ -128,6 +128,15 @@ autotraceinclude_HEADERS=\
 		src/color.h
 
 libautotrace_la_LDFLAGS = -version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE)
+libautotrace_la_LIBADD = \
+		$(GRAPHICSMAGICK_LIBS)		\
+		$(IMAGEMAGICK_LIBS)		\
+		$(LIBPNG_LIBS)		\
+		$(LIBSWF_LIBS)		\
+		$(LIBPSTOEDIT_LIBS)		\
+		$(GLIB2_LIBS)			\
+		$(INTLLIBS)			\
+		-lm
 
 #
 # noinst_HEADERS: headers shared between lib and bin.
@@ -145,15 +154,8 @@ AM_CPPFLAGS = $(GRAPHICSMAGICK_CFLAGS) $
 
 autotrace_LDADD =				\
 		libautotrace.la			\
-		$(GRAPHICSMAGICK_LIBS)		\
-		$(IMAGEMAGICK_LIBS)		\
-		$(LIBPNG_LIBS)		\
-		$(LIBSWF_LIBS)		\
-		$(LIBPSTOEDIT_LIBS)		\
-		$(GLIB2_LIBS)			\
 		$(POPT_LIBS)			\
-		$(INTLLIBS)			\
-		-lm
+		$(nil)
 
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA= autotrace.pc
